openlibrary/core/models.py

```diff
@@ -874,6 +874,14 @@ def is_usergroup_member(self, usergroup):
             usergroup = '/usergroup/%s' % usergroup
         return usergroup in [g.key for g in self.usergroups]
 
+    def is_subscribed_user(self, username):
+        my_username = self.get_username()
+        return (
+            PubSub.is_subscribed(my_username, username)
+            if my_username != username
+            else False
```

Comment:
I understand that `mb.is_subscribed` returns -1  in the event that the users are the same. Here, I opted not to do that, and instead kept the result typed to boolean only, under the assumption that there's a specific use case for `is_subscribed` to use -1, but maybe that's a bad assumption. Should this return -1 in that case instead of False?